datastore/integration_test.go

```diff
@@ -729,30 +730,57 @@ func TestIntegration_AggregationQueries(t *testing.T) {
 		}
 	}()
 
-	baseQuery := NewQuery("SQChild").Ancestor(parent)
+	// Create transaction with read before create
+	txBeforeCreate, err := client.NewTransaction(ctx, []TransactionOption{ReadOnly, WithReadTime(beforeCreate)}...)
+	if err != nil {
+		t.Fatalf("client.NewTransaction: %v", err)
+	}
+
+	// Create transaction with read after create
+	txAfterCreate, err := client.NewTransaction(ctx, []TransactionOption{ReadOnly, WithReadTime(time.Now())}...)
```

Comment:
Is it interesting or useful to make read time something other than `time.Now()`? Should we rename `beforeCreate` to `readTime` and use that again here?

Reply:
Renamed to readTime but still  using time.Now() since I want to get the time after the entities have been created and check for their presence in the result